README.md

```diff
@@ -29,12 +29,12 @@ Routing and networking for Cloud Foundry running on Kubernetes.
     
     ```yaml
     cfroutesync:
-      ccCA: 'path_to_cloud_controller_ca'
-      ccBaseURL: 'https://api.example.com'
-      uaaCA: 'path_to_uaa_ca'
-      uaaBaseURL: 'https://uaa.example.com'
-      clientName: 'uaaClientName'
-      clientSecret: 'uaaClientSecret'
+      ccCA: 'Base64 encoded cloud controller CA certificate'
+      ccBaseURL: 'Base64 encoded https://api.example.com'
+      uaaCA: 'Base64 encoded path UAA CA certificate'
```

Comment:
Can you remove the word "path" from here? The original description about it being a path was wrong. 😞